src/views/ocr/components/OcrResultPanel.vue

```
@@ -0,0 +1,244 @@
+        file: fileData as string,
+        fileType: 1,
+        visualize: true,
+      })
+
+      if (response.errorCode === 0 && response.result) {
+        ocrResults.value = response.result.ocrResults
+        Message.success(`识别成功，共 ${ocrResults.value.length} 页`)
+      } else {
+        Message.error(response.errorMsg || '识别失败')
+      }
+    }
```

ESLint

[error] Closing curly brace does not appear on the same line as the subsequent block.

(style/brace-style)